new-frontiers-14/frontier-station-14
# Replace tiny fans with directional in Anomalous Geode POI

...Also fixed its grid name



## About the PR
Replaced tiny fans with directional fans. Fixed grid name.

## Why / Balance
Compliance and clarity.

## How to test
Load up the grid, look at the new descriptive grid name in the grid list, teleport in, kill the aliens, check out those sweet directional fans on the docks.

## Media
![gridname](https://github.com/user-attachments/assets/961cec96-3405-448f-b41c-a0b8d494b6ab)
![dock1](https://github.com/user-attachments/assets/65b82faf-5237-4134-9d00-a85d4cc56d96)
![dock2](https://github.com/user-attachments/assets/8b16df48-31f8-4efb-9f6a-0f33a7330076)


## Requirements



## Breaking changes
None

**Changelog**

:cl:
- tweak: Updated Anomalous Geode docks to use directional fans
